### PR TITLE
Query operation name

### PR DESCRIPTION
**Pull request by eriiska**
Pass graphql operation name down into queryer

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -43,6 +43,7 @@ github.com/spf13/cobra v0.0.3/go.mod h1:1l0Ry5zgKvJasoi3XT1TypsSe7PqH0Sj9dhYf7v3
 github.com/spf13/pflag v1.0.3 h1:zPAT6CGy6wXeQ7NtTnaTerfKOsV6V6F8agHXFiazDkg=
 github.com/spf13/pflag v1.0.3/go.mod h1:DYY7MBk1bdzusC3SYhjObp+wFpr4gzcvqqNjLnInEg4=
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/objx v0.1.1 h1:2vfRuCMp5sSVIDSqO8oNnWJq7mPa6KVP3iPIwFBuy8A=
```
Comment (bardhi) [inline go.sum]: Is this change intentional and/or related?

Comment (eriiska) [reply go.sum]: Unintentional

**Comment by bk-cribl** [comment]
👍

File: execute_test.go [hunk 2]
```diff
@@ -23,6 +23,17 @@ func (f roundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
...
 }
 
+type testQuery struct {
+	queryer               graphql.Queryer
+	t                     *testing.T
+	expectedOperationName string
+}
+
+func (q *testQuery) Query(ctx context.Context, input *graphql.QueryInput, receiver interface{}) error {
+	assert.Equal(q.t, input.OperationName, q.expectedOperationName)
```
Comment (bardhi) [inline execute_test.go]: Is it better to do this than just do assert.Equal calls inside the test (e.g. like the assertion for hello world bellow on line 1586?

Comment (eriiska) [reply execute_test.go]: Ahh that's what I was missing. Yes that makes more sense

**Comment by bardhi** [comment]
LGTM!
